app/data_definitions/base.rb

```diff
@@ -0,0 +1,25 @@
+module DataDefinitions
+  class Base
+    attr_accessor :document
+
+    def initialize
+      xml_file = format('%s.xml', self.class.name.underscore.split('/').last)
+      xml_path = File.join(Rails.root, 'app/data_definitions/from_cascade', xml_file)
+      @document = Nokogiri::XML(File.read(xml_path)) if File.exist?(xml_path)
+    end
+
+    # rubocop:disable Rails/OutputSafety
+    def get_child_value(field, unescaped=false)
```

Comment:
Added this method to replace field-specific helper methods in data definition classes. It also better echoes what Cascade itself does in velocity templates.